hermes/probe/monitor.go

```diff
@@ -12,9 +12,160 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+// Authors:
+// - Alicja Kwiecinska (kwiecinskaa@google.com) github: alicjakwie
+// - Evan Spendlove, GitHub: evanSpendlove.
+//
 // Monitor implements a probe that monitors a storage system using the
 // Hermes algorithm.
+// Package probe implements the probe that Hermes uses to monitor
```

Comment:
GoDoc expects the "// Package probe ..." on the first line in the file.

Reply:
Ah okay, I think I misunderstood this before. I'll amend this.

Reply:
I don't see the fix.